src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -24,14 +24,16 @@ internal sealed class RuntimeMetrics
 #endif
     private const int NumberOfGenerations = 3;
 
-    private static readonly string[] GenNames = new string[] { "gen0", "gen1", "gen2", "loh", "poh" };
+    private static readonly string[] GenNames = ["gen0", "gen1", "gen2", "loh", "poh"];
+#if NET
```

Comment:
Is this added by some automation tool?

Reply:
Nope, it was manual modification. `isGcInfoAvailable` was marked as unused on some target frameworks. I manually modified it.